daemon/volumes.go

```diff
@@ -287,6 +290,27 @@ func (daemon *Daemon) verifyVolumesInfo(container *Container) error {
 	return container.ToDisk()
 }
 
+func (daemon *Daemon) migrateVolume(id, hostPath string) error {
+	volumePath := filepath.Join(daemon.root, defaultVolumesPathName, id)
+	backupPath := filepath.Join(daemon.root, defaultVolumesPathName, id+".back")
+
+	var err error
+	if err = os.Rename(volumePath, backupPath); err != nil {
```

Comment:
could you maybe add a comment to this flow, i'm trying to follow but having all the vars named `*Path` make it hard to folow.